stories/Button/Button.stories.tsx

```diff
@@ -57,6 +61,24 @@ export const Esempi = () => {
   );
 };
 
+Esempi.storyName = 'Esempi';
+
+export const Personalizzato = () => {
```

Comment:
While declared this has not been imported in the `Button.stories.mdx` file and replaced the inline version.

Reply:
resolved in commit 72123f44ceb3b5bd87baceb804316c16cff88841